packages/vuetify/src/components/VAutocomplete/VAutocomplete.tsx

```diff
@@ -122,9 +122,13 @@ export const VAutocomplete = genericComponent<new <
       props,
       'modelValue',
       [],
-      v => transformIn(wrapInArray(v)),
       v => {
-        const transformed = transformOut(v)
+        const transformVal = revertSymbolIfNull(v)
+        return transformIn(transformVal == null ? [transformVal] : wrapInArray(transformVal))
```

Comment:
If we are no longer supporting undefined should this change?

Reply:
`transformVal == null` is necessary to allow null to be legit. Using `== null` is just consistent with all other places